### ISSUE TITLE
Decide strategy for calculated fields

### ISSUE THREAD

**Issue by cjdd3b** [newsdev/nyt-campfin#27]
Before we implement anything we should:

- [ ] Decide with Sarah which fields we're calculating (same meeting as #14, #26)
- [ ] Decide where calculations live in the codebase. Ideally these will be broken out and documented as individual pieces of logic. Functions?
- [ ] Decide where in the processing pipeline these will fire

**Comment by cjdd3b** [comment]
Note to self: List out all the aggregates Derek is running for reference

**Comment by cjdd3b** [comment]
Here's a couple:

- [ ] [Calculating contribution ranges](https://github.com/newsdev/campaign_finance/blob/2016/lib/itemization_processor.rb#L197-L211)

- [ ] [Calculating primary/general totals](https://github.com/newsdev/campaign_finance/blob/2016/lib/itemization_processor.rb#L197-L211)

**Comment by rshorey** [comment]
- [ ] max-out counts and totals
- [ ] small-donor total incl itemized under 200 and unitemized

**Comment by rshorey** [comment]
@cjdd3b, Sarah said she defers to Nick on contribution ranges, is that a conversation you could have with him, please?

**Comment by rshorey** [comment]
I'm going to stick with that was done before for itemization buckets. Will do this after #87 because it differs for pacs and candidates.

**Comment by rshorey** [comment]
buckets are going to be done in a view, moving that to its own issue and closing this one.

**Comment by cjdd3b** [comment]
Probably model method, not view. But dynamic nonetheless.

**Comment by rshorey** [comment]
ok, so we don't care whether it's queryable in the DB?

**Comment by cjdd3b** [comment]
Oh, a database view, not a Django view. We can do that too if Sarah ends up
needing it, but for the purposes of the app I think model methods are fine.
